hangman/lib/game/game.ex

```diff
@@ -0,0 +1,89 @@
+defmodule Hangman.Game do
+
+#Public Functions
+
+    def new_game() do
```

Comment:
This is one of those good news/bad news comments, but the net result is no change to the grade :)

First, I really like the way you've added `new_game/1`. It makes testing a bunch easier.

However, in doing so you've introduced duplication:

~~~ elixir
def new_game(word) do
     initialize_game( word |> String.split("", trim: true) )
end
~~~

and

~~~ elixir
defp retrieve_word() do
     Dictionary.WordList.random_word() |> String.split( "", trim: true ) 
end
~~~

Both functions have knowledge of the conversion from a string into the internal representation.

I feel it would have been better to have

~~~ elixir
def new_game() do
  Dictionary.random_word() |> new_game()
end

def new_game(word) do
  initialize_game( word |> String.split("", trim: true) )
end
~~~

Reply:
Thanks for spotting that! That makes perfect sense